social-network-api/src/main/java/com/facebook/dto/friends/FriendsResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.facebook.dto.friends;
+
+import com.facebook.model.friends.FriendsStatus;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+public class FriendsResponse {
+
+    private Long id;
+
+    private LocalDateTime created_date;
+
+    private LocalDateTime last_modified_date;
+
+    private Long userId;
+
+    private Long friendId;
+
+    private FriendsStatus status;
```

Comment:
Why the fields are in different style? Please, use camelCase for fields, if you need to get snake_case in the json you can add the annotation @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)

Reply:
Fixed)